third-party/genuine-fantasy-press/forgotten-secrets/invocations.xml

```diff
@@ -4,396 +4,338 @@
 		<name>Invocations</name>
 		<description>COFSA Invocations</description>
 		<author url="www.genfantasypress.com">Genuine Fantasy Press</author>
-		<update version="0.0.2">
+		<update version="0.0.4">
 			<file name="invocations.xml" url="https://raw.githubusercontent.com/aurorabuilder/elements/master/third-party/genuine-fantasy-press/forgotten-secrets/invocations.xml" />
 		</update>
 	</info>
 	
 	<element name="The Archivist’s Journal" type="Class Feature" source="The Compendium of Forgotten Secrets" id="ID_GFP_COFSA_CLASS_FEATURE_ELDRITCH_INVOCATION_THE_ARCHIVISTS_JOURNAL">
 		<supports>Eldritch Invocation</supports>
-		<prerequisite>Accursed Archive patron, Pact of the Tome</prerequisite>
-		<requirements>ID_GFP_COFSA_ARCHETYPE_OTHERWORLDLY_PATRON_ACCURSED_ARCHIVE,  ID_WOTC_PHB_CLASS_FEATURE_WARLOCK_PACT_BOON_PACT_OF_THE_TOME</requirements>
+		<prerequisite>Accursed Archive Patron, Pact of the Tome</prerequisite>
```

Comment:
when not part of the name, these are lowercase
you can however add _feature_ after _Pact of the Tome_